instances/treasury-testing.near/widget/config/data.jsx

```diff
@@ -46,4 +46,5 @@ return {
     </svg>
   ),
   themeColor: "#05a36e",
+  allowLockupCancellation: true,
```

Comment:
Ori mentioned to also include it for `treasury-devdao`, please add to their config as well.